components/script_bindings/codegen/CodegenRust.py

```diff
@@ -62,12 +62,64 @@
 CONSTRUCT_HOOK_NAME = '_constructor'
 HASINSTANCE_HOOK_NAME = '_hasInstance'
 
-RUST_KEYWORDS = {"abstract", "alignof", "as", "become", "box", "break", "const", "continue",
-                 "else", "enum", "extern", "false", "final", "fn", "for", "if", "impl", "in",
-                 "let", "loop", "macro", "match", "mod", "move", "mut", "offsetof", "override",
-                 "priv", "proc", "pub", "pure", "ref", "return", "static", "self", "sizeof",
-                 "struct", "super", "true", "trait", "type", "typeof", "unsafe", "unsized",
-                 "use", "virtual", "where", "while", "yield"}
+RUST_KEYWORDS = {
+    "abstract",
+    "alignof",
+    "as",
+    "async",
+    "await",
+    "become",
+    "box",
+    "break",
+    "const",
+    "continue",
+    "crate",
+    "do",
+    "dyn",
+    "else",
+    "enum",
+    "extern",
+    "false",
+    "final",
+    "fn",
```

Comment:
This is the formatting `ruff` gave me. Seems odd, maybe it didn't pick up a config file something. We don't actually enforce the formatting yet (https://github.com/servo/servo/pull/37117)